src/components/Dialog.scss

```diff
@@ -1,6 +1,11 @@
 @import "../css/_variables";
 
 .excalidraw {
+  .Dialog {
+    user-select: text;
+    cursor: auto;
```

Comment:
isn't that the default one?

Reply:
not if we override it from upstream (when changing cursor on scene)